### PR TITLE
[infra] Set external mirror server login

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit adds reading environment variable for mirror server login.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
It will resolve build fail 12094 ~ 12098

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

File: infra/buildtool/CMakeLists.txt [hunk 1]
```diff
@@ -20,13 +20,26 @@ endmacro(nnas_include)
 
 nnas_include(OptionTools)
 envoption(EXTERNAL_DOWNLOAD_SERVER "https://github.com")
+# Sync with traditional external download tool: server login
+# TODO Migrate external download tool
+envoption(EXTERNAL_SERVER_USERPWD "")
+if(EXTERNAL_SERVER_USERPWD)
+  string(REGEX MATCH "^[^:]*" EXTERNAL_SERVER_USERNAME ${EXTERNAL_SERVER_USERPWD})
+  string(REGEX MATCH "[^:]*$" EXTERNAL_SERVER_PASSWORD ${EXTERNAL_SERVER_USERPWD})
```
Comment (seanshpark) [inline infra/buildtool/CMakeLists.txt:28]: don't know much about this but this is to cut `USER`, `PWD` from `USER:PWD`  ?

Comment (hseok-oh) [reply infra/buildtool/CMakeLists.txt:28]: Right.

**Comment by hseok-oh** [comment]
Merge before all test passed. (x64 passed)

**Pull request merged**